src/Nri/Ui/Button/V8.elm

```diff
@@ -0,0 +1,783 @@
+module Nri.Ui.Button.V8 exposing
+    ( ButtonSize(..), ButtonWidth(..), ButtonStyle(..), ButtonState(..)
+    , button
+    , delete
+    , toggleButton
+    , link, linkSpa
+    , linkExternal, linkWithMethod, linkWithTracking, linkExternalWithTracking
+    )
+
+{-|
+
+
+# Changes from V7:
```

Comment:
I wanna put :tada: on all four bullet points below but that would be too much, so instead you get:

- :tada:
- :tada:
- :tada:
- :tada: